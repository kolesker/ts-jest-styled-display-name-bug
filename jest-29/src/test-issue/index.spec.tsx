import React from 'react';
import { mount } from 'enzyme';


import Component from './index';


describe('Component', () => {

  it('it should match mount snapshot', () => {
    const wrapper = mount(<Component />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
