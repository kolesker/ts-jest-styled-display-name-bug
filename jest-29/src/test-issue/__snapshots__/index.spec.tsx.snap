// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component it should match mount snapshot 1`] = `
"<Component>
  <StyledExample>
    <div className=\\"sc-beySbM AIKwo\\" />
  </StyledExample>
</Component>"
`;
